packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.stories.tsx

```diff
@@ -138,36 +139,81 @@ export const HasMetRequirements: Story = {
   },
 }
 
-export const HasUnmetRequirements: Story = {
+const expandedMockAccessRequirements = [
+  ...mockAccessRequirements,
+  { ...mockManagedACTAccessRequirement, id: 21 },
+  { ...mockToUAccessRequirement, id: 22 },
+
+  { ...mockSelfSignAccessRequirement, id: 23 },
+  { ...mockACTAccessRequirement, id: 24 },
```

Comment:
The mock access requirements are a mixed bag of types, where the first set have approvals and the second set do not.  The output is now sorted by AR type, and within each type they are sorted by approval status. 
We will never see this kind of mess outside of a testing environment (all types of ARs, where each type has both met and unmet represented).